frontend/packages/client/src/components/Label.js

```diff
@@ -3,7 +3,7 @@ import classnames from 'classnames';
 
 export default function Label({ labelText = '', classNames } = {}) {
   const className = classnames(
-    'has-background-black has-text-white rounded-sm px-3 py-2 mr-2',
+    'has-background-black has-text-white rounded-lg px-3 py-2',
```

Comment:
removed `mr-2` from here as it wasn't needed in every instance and should be defined in the layout that includes `Label`